x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/services/analytics_service/delete_analytics.ts

```diff
@@ -39,3 +37,114 @@ export const deleteAnalytics = async (d: DataFrameAnalyticsListRow) => {
+    }
+    const status = await ml.dataFrameAnalytics.deleteDataFrameAnalyticsAndTargetIndex(
+      d.config.id,
+      deleteTargetIndex,
+      deleteTargetPattern
+    );
+    if (status.analyticsJobDeleted?.success) {
+      toastNotifications.addSuccess(
+        i18n.translate('xpack.ml.dataframe.analyticsList.deleteAnalyticsSuccessMessage', {
+          defaultMessage: 'Request to delete data frame analytics {analyticsId} acknowledged.',
+          values: { analyticsId: d.config.id },
+        })
+      );
+    }
+    if (status.analyticsJobDeleted?.error) {
+      const error = extractErrorMessage(status.targetIndexDeleted.error);
+      toastNotifications.addDanger(
+        i18n.translate('xpack.ml.dataframe.analyticsList.deleteAnalyticsErrorMessage', {
+          defaultMessage:
+            'An error occurred deleting the data frame analytics {analyticsId}: {error}',
+          values: { analyticsId: d.config.id, error },
+        })
+      );
+    }
+
+    if (status.targetIndexDeleted?.success) {
+      toastNotifications.addSuccess(
+        i18n.translate('xpack.ml.dataframe.analyticsList.deleteAnalyticsWithIndexSuccessMessage', {
+          defaultMessage: 'Request to delete destination index {destinationIndex} acknowledged.',
+          values: { destinationIndex },
+        })
+      );
+    }
+    if (status.targetIndexDeleted?.error) {
+      const error = extractErrorMessage(status.targetIndexDeleted.error);
+      toastNotifications.addDanger(
+        i18n.translate('xpack.ml.dataframe.analyticsList.deleteAnalyticsWithIndexErrorMessage', {
+          defaultMessage:
+            'An error occurred deleting destination index {destinationIndex}: {error}',
+          values: { destinationIndex, error },
+        })
+      );
+    }
+
+    if (status.targetIndexPatternDeleted?.success) {
+      toastNotifications.addSuccess(
+        i18n.translate(
+          'xpack.ml.dataframe.analyticsList.deleteAnalyticsWithIndexPatternSuccessMessage',
+          {
+            defaultMessage: 'Request to delete index pattern {destinationIndex} acknowledged.',
+            values: { destinationIndex },
+          }
+        )
+      );
+    }
+    if (status.targetIndexPatternDeleted?.error) {
+      const error = extractErrorMessage(status.targetIndexPatternDeleted.error);
+      toastNotifications.addDanger(
+        i18n.translate(
+          'xpack.ml.dataframe.analyticsList.deleteAnalyticsWithIndexPatternErrorMessage',
+          {
+            defaultMessage: 'An error occurred deleting index pattern {destinationIndex}: {error}',
+            values: { destinationIndex, error },
+          }
+        )
+      );
+    }
+  } catch (e) {
+    toastNotifications.addDanger(
+      i18n.translate('xpack.ml.dataframe.analyticsList.deleteAnalyticsErrorMessage', {
+        defaultMessage:
+          'An error occurred deleting the data frame analytics {analyticsId}: {error}',
```

Comment:
As above, probably worth adding `job` in the text - `An error occurred deleting the data frame analytics job {analyticsId}: {error}`.

Reply:
Done in https://github.com/elastic/kibana/pull/66934/commits/5bd97e9c466ad1a88d32da4f432583d5ddc3aed0